### PR TITLE
Hide problem instances for competition

### PR DESCRIPTION
**Pull request by ThijsEgberts**
Hide problem instances in the frontend for competition problems

### CODE REVIEW

File: frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx [hunk 1]
```diff
@@ -57,13 +57,15 @@ function ProblemOccurrenceOverviewPage() {
     setCurrentTab(e.target.id);
   }
 
+  console.log(problemData)
```
Comment (Booksh) [inline frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx:60]: Is this line meant to stay in there or just leftover testing?

Comment (ThijsEgberts) [reply frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx]: nope, I forgot to remove it

File: frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx [hunk 2]
```diff
@@ -97,11 +97,16 @@ function ProblemOccurrenceOverviewPage() {
...
-            <li className="tab-selector-item">
-              <a role="button" className={currentTab == "4" ? "active tab-selector-link": "tab-selector-link"} /*active={currentTab == "4"}*/ id="4" onClick={handleTabSwitch}>
-                Problem instances
-              </a>
-            </li>
+            
+            {/* Hide this tab if the problem is a competition style problem */}
+            {problemData.category.style != 0 ? 
+              <li className="tab-selector-item">
+                <a role="button" className={currentTab == "4" ? "active tab-selector-link": "tab-selector-link"} /*active={currentTab == "4"}*/ id="4" onClick={handleTabSwitch}>
```
Comment (vlhtg) [inline frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx:104]: isn't this a bit long?

**Review by vlhtg — APPROVED** [review]
yea, I'll approve it. Don't tell Teun tho

File: frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx (hunk 2 above)
Comment (ThijsEgberts) [reply frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx:104]: no

Comment (vlhtg) [reply frontend/src/components/problemOccurenceOverview/ProblemOccurrenceOverviewPage.jsx:104]: anyway

**Pull request merged**